include/details.md

```diff
@@ -42,18 +44,24 @@ __Documentation__
     - {{ feature }}
     {% endfor %}
 
-{% if (features.programming_languages or features.environments) %}
-=== "Kernels"
+{% if (features.programming_languages or features.environments) %} 
+=== "Kernels :material-information-outline:{ title="Links to Kernels" }" 
     {% if features.programming_languages %}
     Programming Languages:
     {% for kernel in features.programming_languages %}
+    {% if Links[kernel | lower] %}
+    - [{{ kernel }}]({{ Links[kernel | lower] }})
+    {% else %}
     - {{ kernel }}
-    {% endfor %}
     {% endif %}
-    {% if features.environments %}
+    {% endfor %}
+{% endif %}
+    {% if features.environments %} 
     Environments:
     {% for kernel in features.environments %}
-    - {{ kernel }}
+-   {{ kernel }} {% if Links[kernel] %} [{{ Links[kernel] }}]({{ Links[kernel] }}) 
+    {% else %}  
+    {% endif %}
```

Comment:
Compare to if else loop of features.programming_language. The else-part contains nothing at the moment...